src/pages/MetricDetail.svelte

```diff
@@ -219,6 +220,11 @@
           <HelpHoverable
             content={helpText.searchFox.text}
             link={helpText.searchFox.link} />
+          {#if isExpired(metric.expires)}
+            <HelpHoverable content={'This metric has expired.'}>
+              <WarningIcon width={'15px'} height={'15px'} />
```

Comment:
It would be nice to not hardcode this here, but maybe that's a problem to solve another time.

Reply:
It was not possible to pass these values down as props in `HelpHoverable`. I hope that this use case is rare enough that hard coding is okay, for now.